### ISSUE TITLE
Looks like ./ansible/99-destroy-cluster.yml doesn't work well anymore.

### ISSUE THREAD

**Issue by rbo** [RedHat-EMEA-SSA-Team/hetzner-ocp4#265]
After running `ansible-navigator run ./ansible/99-destroy-cluster.yml`

The VM's are still there:
```
[root@pluto ~]# virsh list --all
 Id   Name             State
---------------------------------
 -    demo-bootstrap   shut off
 -    demo-compute-0   shut off
 -    demo-compute-1   shut off
 -    demo-master-0    shut off
 -    demo-master-1    shut off
 -    demo-master-2    shut off
```

**Comment by rbo** [comment]
Fix merged with #267 into devel and will be merged into master with #252

**Comment by rbo** [comment]
Ups looks like not solved...

**Comment by rbo** [comment]
```
 podman run -ti --rm quay.io/redhat-emea-ssa-team/hetzner-ocp4-ansible-ee:202304141234 ansible-galaxy collection list

# /usr/share/ansible/collections/ansible_collections
Collection             Version
---------------------- -------
amazon.aws             5.4.0  
ansible.posix          1.5.2  
azure.azcollection     1.15.0 
community.aws          5.4.0  
community.azure        2.0.0  
community.crypto       2.11.1 
community.digitalocean 1.23.0 
community.dns          2.5.2  
community.general      6.5.0  
community.kubernetes   2.0.1  
community.libvirt      1.2.0  
google.cloud           1.1.3  
kubernetes.core        2.4.0  

podman run -ti --rm quay.io/redhat-emea-ssa-team/hetzner-ocp4-ansible-ee:master ansible-galaxy collection list

# /usr/share/ansible/collections/ansible_collections
Collection             Version
---------------------- -------
amazon.aws             3.3.0  
ansible.posix          1.4.0  
azure.azcollection     1.13.0 
community.aws          3.3.0  
community.azure        1.1.0  
community.crypto       2.3.3  
community.digitalocean 1.20.0 
community.general      5.1.1  
community.kubernetes   2.0.1  
community.libvirt      1.1.0  
google.cloud           1.0.2  
kubernetes.core        2.3.2  

```

Rebuild make sense, but force is not invluded in community.libvirt 1.2 

https://github.com/ansible-collections/community.libvirt/issues/40 - Fixed August 12th
community.libvirt 1.20 release created August 2nd

**Comment by rbo** [comment]
Installed `community.libvirt` from source:
```yaml
  - name: https://github.com/ansible-collections/community.libvirt
    type: git
    version: main
```

**Comment by rbo** [comment]
```
podman run -ti --rm quay.io/redhat-emea-ssa-team/hetzner-ocp4-ansible-ee:202304141355 ansible-galaxy collection list

# /usr/share/ansible/collections/ansible_collections
Collection             Version
---------------------- -------
amazon.aws             5.4.0  
ansible.posix          1.5.2  
azure.azcollection     1.15.0 
community.aws          5.4.0  
community.azure        2.0.0  
community.crypto       2.11.1 
community.digitalocean 1.23.0 
community.dns          2.5.2  
community.general      6.5.0  
community.kubernetes   2.0.1  
community.libvirt      1.2.1  
google.cloud           1.1.3  
kubernetes.core        2.4.0  
```

**Comment by rbo** [comment]
Finally fixed with #268
